plugins/node/opentelemetry-instrumentation-pino/src/instrumentation.ts

```diff
@@ -72,6 +72,11 @@ export class PinoInstrumentation extends InstrumentationBase {
             return pinoModule(...(args as Parameters<Pino>));
           }, pinoModule);
 
+          // @ts-expect-error the same function reexported
+          patchedPino.pino = patchedPino;
```

Comment:
as far as I remember these two exports are not present in all versions so maybe use an if around like this:
```ts
if (typeof(pinoModule.pino) === "function") {
  pinoModule.pino = patchedPino;
}
```

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1108/commits/a8700762c89367abfb706602bcf82cbd84f438f4